guide/message-processor-service/message_processor.bal

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mb;
+
+documentation { Define the message queue endpoint for sending newly processed new information messages. }
+endpoint mb:SimpleQueueSender queueProcessedQueue {
+    host: "localhost",
+    port: 5672,
+    queueName: "ProcessedQueue"
+};
+documentation { Attributes associated with the service endpoint. }
+endpoint http:Listener httpListner {
+    port: 8080
+};
+documentation {  via HTTP/1.1. }
+@http:ServiceConfig {
+    basePath: "/"
+}
+service<http:Service> MessageProcessingService bind httpListner {
+    @http:ResourceConfig {
+        methods: ["POST"], consumes: ["application/json"], produces: ["plain/text"],
+        path: "/activity"
+    }
+    documentation { Resources for reciving geo activities captured from the sensor }
```

Comment:
reciving -> receiving

Reply:
Fix typo reciving -> receiving in every places.